src/app/app.ts

```diff
@@ -5,7 +5,8 @@ import * as cors from '@koa/cors'
 import menuRouter from '../routes/menu.routes'
 import userRouter from '../routes/user.routes'
 import cartRouter from '../routes/cart.routes'
-import orderRouter from '../routes/order.routes'
+import bookingRouter from "../routes/booking.routes";
```

Comment:
As far as I see, you are not using semicolons at all. This also applies to the rest of the files, stick to the same style guide (eslint will handle this)